src/backend/base/langflow/base/tools/constants.py

```diff
@@ -35,6 +35,14 @@
         "edit_mode": EditMode.INLINE,
         "hidden": True,
     },
+    {
+        "name": "status",
+        "display_name": "Status",
+        "type": "boolean",
+        "description": "Indicates whether the tool is currently active. Set to True to activate this tool.",
+        "default": "False",
```

Comment:
Should this be set to `True` by default instead? I think it shouldn't have quotes either.